satellite/audit/reporter_test.go

```diff
@@ -58,8 +56,6 @@ func TestRecordAuditsAtLeastOnce(t *testing.T) {
 	testplanet.Run(t, testplanet.Config{
 		SatelliteCount: 1, StorageNodeCount: 1, UplinkCount: 0,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
-		err := planet.Satellites[0].Audit.Service.Close()
```

Comment:
we should still close the audit worker. Also, if we make `Reporter` exported on `peer.Audit`, we do not necessarily need to create a new reporter as part of this test (and other tests in this file).

Reply:
I think at least for the tests in this file, it is still necessary to make the new reporter because we're instantiating it with different max retry values, which affects the outcome of the test.